onnxruntime/contrib_ops/cpu/qlinear_binary_op.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+#include "core/util/math_cpuonly.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+template <typename T>
+class QLinearAdd final : public OpKernel {
+ public:
+  QLinearAdd(const OpKernelInfo& info) : OpKernel(info) {
```

Comment:
Should it just be {} at the end? Other kernels don't split this when the constructor is empty.

Reply:
Fixed!